src/main/java/io/vertx/gsoc2018/qotd/QuoteOfTheDayVerticle.java

```diff
@@ -33,7 +35,47 @@ public void start(Future<Void> startFuture) throws Exception {
     HttpServer server = vertx.createHttpServer();
     Router router = Router.router(vertx);
 
-    router.route("/quotes").handler(routingContext -> {
+    router.post(QUOTES_PATH).handler(routingContext -> {
+      HttpServerResponse response = routingContext.response();
+      routingContext.request().bodyHandler(requestBody -> {
+        JsonObject request = requestBody.toJsonObject();
+        String text = request.getString("text");
+        if (text == null) {
+          response.setStatusCode(404);
+          response.end("json in a POST request should have a 'text' filed");
+          return;
+        }
+
+        String author = request.getString("author");
+        if (author == null) {
+          author = DEFAULT_AUTHOR_VALUE;
+        }
+
+        String finalAuthor = author;
+        jdbcClient.getConnection(getConn -> {
+          if (getConn.succeeded()) {
+            SQLConnection connection = getConn.result();
+            String sqlToExecute = String.format("INSERT INTO quotes (text,author) VALUES ('%s', '%s');", text, finalAuthor);
```

Comment:
this is subject to sql injection attacks. it's better to user `connection.updateWithParams`

Reply:
Agree